src/utils/URLUtil.js

```diff
@@ -0,0 +1,44 @@
+var util = require('./Util');
+var parseUri = require('parseUri');
+var containerStore = require('../stores/ContainerStore');
+
+module.exports = {
```

Comment:
@diogomonica here's the updated method with annotations, I pulled it out of the main app.js so I could unit test it. How does it look?